### PR TITLE
Fixing styles for FeedScreen and ContentScreen.

### PR DESCRIPTION
**Pull request by AndMkh7**

### CODE REVIEW

File: screens/Content/ContentScreen.jsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import {useContext} from 'react';
+import React, {useContext} from 'react';
```
Comment (M7Arman) [inline screens/Content/ContentScreen.jsx:1]: why you need this?

**Review by M7Arman — APPROVED** [review]

File: screens/Content/ContentScreen.jsx (hunk 1 above)
Comment (AndMkh7) [reply screens/Content/ContentScreen.jsx:1]: Two days ago I removed the React import, but after installing ESLint I got a warning like this : 
"ESLint: 'React' must be in scope when using JSX (react/react-in-jsx-scope) " . 

After importing React, the warning disappears.

Comment (M7Arman) [reply screens/Content/ContentScreen.jsx:1]: Which React version do you use? If it's one of the last versions, then you should update(specify React version) eslint

Comment (AndMkh7) [reply screens/Content/ContentScreen.jsx:1]: I checked the versions of ESLint (8.41.0) and React JS (18.2.0) and at the moment these versions are the latest for both . Advise, please, does it make sense to update something?

**Pull request merged**